src/w3d/lib/coord.h

```diff
@@ -155,91 +117,19 @@ class Coord3D
         z = 0.0f;
     }
 
-    Coord3D &operator+=(const Coord3D &rhs)
```

Comment:
Revert all of these. Overloaded operators are much much simpler to understand and harder to mess up

Reply:
I added based on what Vector2 and Vector3 have and used them where appropriate.